cas-server-support-saml-googleapps/src/main/java/org/jasig/cas/support/saml/authentication/principal/GoogleAccountsServiceFactory.java

```diff
@@ -63,7 +63,7 @@ public GoogleAccountsServiceFactory() {}
 
     /**
      * Init public and private keys.
-     * @throws Exception if key creation encountered an error.
+     * @throws RuntimeException if key creation encountered an error.
```

Comment:
Another IDE grumpy point. Also, this is something that makes my skin crawl... Nothing should be throwing or catching `Exception`. Imho, that includes documentation. I'm going to keep this stuff to files I otherwise touch and put the rest in another PR.